### DISCUSSION TITLE
(Bug) Friendly stars (npc-97/npc-198) are counted in warp's star counter

### DISCUSSION THREAD

**Discussion by P-Star7** [Wohlstand/TheXTech#389] [General]
Seen in ASMBXT, this is useless for the player because they cannot collect those stars. Friendly star NPCs should be excluded from the count.

**Comment by Wohlstand** [comment]
Levels saved by the old Editor will give this bug. You can re-save all files of an episode using the new Editor or PGE Maintainer to fix up all levels with improper stars counted.
